rpmlb/builder/koji.py

```diff
@@ -136,10 +136,12 @@ def build(self, package_dict, **kwargs):
             epel=self.epel,
         )
 
-        self._add_package(name='{collection}-{name}'.format(
-            collection=self.collection,
-            name=package_dict['name'],
-        ))
+        if package_dict['name'] == self.collection:  # metapackage
```

Comment:
Make this a function. You can then test the function isolated without all the bloody mocking.

Reply:
Yeah, bit of a brainf*rt on my part. I will take time to do this properly next week.

Thanks for keeping the technical debt at bay ☺ !